build.gradle

```diff
@@ -59,10 +59,10 @@ dependencies {
     force = true
   }
   compile 'com.google.code.gson:gson:2.7'
-  if (System.getenv().get('CI_MODE') == 'Nightly') {
-    compile 'com.google.javascript:closure-compiler:1.0-SNAPSHOT'
-  } else {
+  if (System.getenv().get('CI_MODE') == 'Released') {
```

Comment:
This will look surprising and like an error to someone not familiar with google. Add a comment, or edit the README.md.

Reply:
Done.